compliance_checker/protocols/zarr.py

```diff
@@ -6,7 +7,11 @@
 
 from compliance_checker.protocols import netcdf
 
-#
+
+def _fix_windows_slashes(zarr_url):
+    if platform.system() == "Windows":
+        zarr_url = zarr_url.replace("///", "//")
```

Comment:
This one had me chasing red herrings all over the code base. I'm not sure if nczarr has a bug, or Python as_uri, or something else. All I know is that we can't have 3 / on Windows, only 2!